.github/CODEOWNERS

```diff
@@ -12,9 +12,12 @@ src/* @mikiquantum @mustermeiszer @branan @NunoAlexandre
 # Changes to chain-specs
 res/* @mikiquantum @mustermeiszer @branan @NunoAlexandre
 
+# Changes to toml files
+*.toml @mikiquantum @mustermeiszer @branan @NunoAlexandre @lemunozm @wischli
```

Comment:
I think with this line, we can avoid having a lot of reviewers each time our dependencies change. Add yourself if you're not in the list and feel comfortable with rust dependencies/features